src/eureka/S6_planet_spectra/s6_spectra.py

```diff
@@ -733,6 +764,143 @@ def load_s5_saves(meta, log, fit_methods):
+                     'fitted parameters')
+        log.writelog(f'  Skipping {y_param}')
+        return meta
+    n_samples = len(a0[0])
+
+    # Load a1 string coefficients
+    meta.y_param = 'a1'+suffix
+    a1 = load_s5_saves(meta, log, fit_methods, n_samples=n_samples)
+    if all(np.all(v == 0) for v in a1):
+        # The parameter could not be found - assume fixed to 0
+        log.writelog(f'  Parameter {meta.y_param} was not in the list of '
+                     'fitted parameters, assumed to be 0.')
+
+    # Load b1 string coefficients
+    meta.y_param = 'b1'+suffix
+    b1 = load_s5_saves(meta, log, fit_methods, n_samples=n_samples)
+    if all(np.all(v == 0) for v in b1):
+        # The parameter could not be found - assume fixed to 0
+        log.writelog(f'  Parameter {meta.y_param} was not in the list of '
+                     'fitted parameters, assumed to be 0.')
+
+    # Load a2 string coefficients
+    meta.y_param = 'a2'+suffix
+    a2 = load_s5_saves(meta, log, fit_methods, n_samples=n_samples)
+    if all(np.all(v == 0) for v in a2):
+        # The parameter could not be found - assume fixed to 0
+        log.writelog(f'  Parameter {meta.y_param} was not in the list of '
+                     'fitted parameters, assumed to be 0.')
+
+    # Load b2 string coefficients
+    meta.y_param = 'b2'+suffix
+    b2 = load_s5_saves(meta, log, fit_methods, n_samples=n_samples)
+    if all(np.all(v == 0) for v in b2):
+        # The parameter could not be found - assume fixed to 0
+        log.writelog(f'  Parameter {meta.y_param} was not in the list of '
+                     'fitted parameters, assumed to be 0.')
+
+    # Load a3 string coefficients
+    meta.y_param = 'a3'+suffix
+    a3 = load_s5_saves(meta, log, fit_methods, n_samples=n_samples)
+    if all(np.all(v == 0) for v in a3):
+        # The parameter could not be found - assume fixed to 0
+        log.writelog(f'  Parameter {meta.y_param} was not in the list of '
+                     'fitted parameters, assumed to be 0.')
+
+    # Load b3 string coefficients
+    meta.y_param = 'b3'+suffix
+    b3 = load_s5_saves(meta, log, fit_methods, n_samples=n_samples)
+    if all(np.all(v == 0) for v in b3):
+        # The parameter could not be found - assume fixed to 0
+        log.writelog(f'  Parameter {meta.y_param} was not in the list of '
+                     'fitted parameters, assumed to be 0.')
+
+    # Reset meta.y_param
+    meta.y_param = y_param
+
+    meta.spectrum_median = []
+    meta.spectrum_err = []
+
+    # Convert full angle in degrees to half angle in radians
+    # e.g., an angle of 60 degrees spans -30 to +30 degrees
+    rad = meta.strings_angle*np.pi/360
+    if limb == 'morning':
+        theta = np.linspace(-rad, rad, 100)
+    elif limb == 'evening':
+        theta = np.linspace(np.pi-rad, np.pi+rad, 100)
+    else:
+        # Future development for custom angular range
+        # North, South poles?
+        return meta
```

Comment:
Probably want to add a log statement here explaining why no plots were made here

Reply:
I just deleted the statement for now.